File: 7kyu/tripleX.js
// return true if the first instance of "x" in the string is immediately followed by the string "xx"

function tripleX(str) {
    let idxOfX = str.indexOf("x");
    let threeChars = str.slice(idxOfX, idxOfX+3);
    console.log(threeChars)

    return threeChars === "xxx";
}

console.log(tripleX("abraxxxcadabra")) // true

File: 7kyu/tripleX.py
# return true if the first instance of "x" in the string is immediately followed by the string "xx"

# find() vs index() - index() method raises an exception if no values was found


def triple_x(str):
    idxOfX = str.find("x")
    three_chars = str[idxOfX : idxOfX + 3]
    return three_chars == "xxx"


print(tripleX("abraxxxcadabra"))  # true
